cmd/ak/cmd/sessions/test.go

```diff
@@ -31,6 +31,9 @@ var (
 
 	// File "/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/concurrent/futures/_base.py", line 401, in __get_result`
 	pyLibRe = regexp.MustCompile(`File ".*/lib/python3\.\d+/(.*\.py)", line (\d+), in (.*)`)
+
+	// Some python version like to put an annoying ^^^^ marker to show where the error is.
+	pyAnnoyingErrorLocationMarkerRe = regexp.MustCompile(`^\s*~*\^+$`)
```

Comment:
What's the meaning of (0 or more) `~` characters here?

Reply:
soemtimes it likes to do ~~~~ or some thing donno why

Reply:
I never saw that, but in that case I guess a more accurate regex might be:
```
`^\s*[~^]+$`
```
(Be careful: `[~^]` means "`~` or `^`", but `[^~]` means "everything except `~`")